manifest.json

```diff
@@ -36,6 +36,13 @@
           "label": "link_color_label",
           "value": "#1F73B7"
         },
+        {
+          "identifier": "visited_link_color",
+          "type": "color",
+          "description": "visited_link_color_description",
+          "label": "visited_link_color_label",
+          "value": "#4B61C3"
```

Comment:
@kaspersorensen @augustocravosilva should we also update this value to be the current default, since this is also a new setting?

The current default depends on the [link color of each theme](https://github.com/zendesk/copenhagen_theme/pull/248/files#diff-bc9dff5ad00c2491d2b9619883dea5d5e508fec2aebbf3a77d70087d2cdafd4bL10) though 🤔

Reply:
I think this one is much less intrusive. I would be fine defending this change to customers who complain. The fix for them is really easy and it's "just" the color of visited links, not as visible a change as the brand name showing up next to the logo.

Reply:
Yeah, i was wondering if this setting is even necessary. I don't think many, if any, existing customers using the cph theme would notice.
Alternatively, I would suggest this setting could be a range [0-50] as the % of darkness compared to the normal link color. this would allow existing customers to keep it at 20% darkness and the relation between the link color and visited color would be kept every time they change the link color.

Reply:
We don't want it to be tied to the normal link color. It's much more flexible to have it independent.